rosplan_planning_system/include/rosplan_planning_system/PlanDispatch/EsterelPlanDispatcher.h

```diff
@@ -61,6 +65,10 @@ namespace KCL_rosplan
 
 		bool display_edge_type_;
 
+        /* dispatch flags for robust experiment */
+        bool timeout_actions;
+        double action_timeout_fraction;
```

Comment:
Thew parameters which set these flags should also be included as arguments in the launch file.